pkg/cmdlets/commit/commit.go

```diff
@@ -8,13 +8,18 @@ import (
 	"github.com/go-openapi/strfmt"
 
 	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
 	"github.com/ActiveState/cli/internal/multilog"
 	"github.com/ActiveState/cli/internal/output"
 	gmodel "github.com/ActiveState/cli/pkg/platform/api/graphql/model"
 	"github.com/ActiveState/cli/pkg/platform/api/mono/mono_models"
 	"github.com/ActiveState/cli/pkg/platform/model"
 )
 
+// AuthorDeleted is the placeholder username to use if an author has since been deleted from an
+// organization and their username can no longer be obtained.
+const AuthorDeleted = "<deleted>"
```

Comment:
```suggestion
const AuthorDeleted = locale.Tl("deleted_username", "<deleted>")
```

Reply:
Okay, but this will have to be inside a function since it's not available at compile time. As a result, this constant will be removed and any outside packages that care about this value will need to use the locale module to get it.